apps/web/prisma/schema.prisma

```diff
@@ -136,10 +141,73 @@ model EmailAccount {
   emailTokens    EmailToken[]
   knowledge      Knowledge[]
   chats          Chat[]
+  digests        Digest[]
 
   @@index([lastSummaryEmailAt])
 }
 
+enum DigestStatus {
```

Comment:
move enums to the bottom of the file (to be consistent with how the rest of the file is structured)